test/step-1.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {State} from "../src/state";
+import {ShipmentIdGenerator} from "../src/shipment-id-generator";
+import {Shipment} from "../src/shipment";
+import {IShipper} from "../src/strategy/shipper";
+import {Context} from "../src/strategy/context";
+
+
+describe('Shipment', () => {
+  it('ship', () => {
+    const state = new State(
+        ShipmentIdGenerator.getShipmentId(),
+        'Minsk, Esenina 123',
+        'Wroclaw, Tadeusza 143',
+        '20345',
+        '40549',
+        100
+    )
+    const context = new Context();
+    const shipment = new Shipment(state, context);
```

Comment:
Just another option
Since you have a state here you could also setStrategy directly on shipment instance like this:

```
const shipment = new Shipment(state);
shipment.setShipperStrategy(defineStrategy(state.fromZipCode))
```